packages/db/src/repositories/competition.ts

```diff
@@ -1865,6 +1865,12 @@ export class CompetitionRepository {
 
       if (params.sort) {
         dataQuery = getSort(dataQuery, params.sort, competitionOrderByFields);
+      } else {
```

Comment:
here's the only logic change: default to start date or created at. this ensure the `/competitions` response won't be a random order (i.e., whatever Postgres decides).

Reply:
nitpick, but IMO this type of default-handling logic belongs at the service layer, not the repository layer.  @asutula and I have been working on getting business logic out of the controllers and repositories as much as possible